simapp/utils.go

```diff
@@ -500,7 +497,11 @@ func GenStakingGenesisState(
 
 // GetSimulationLog unmarshals the KVPair's Value to the corresponding type based on the
 // each's module store key and the prefix bytes of the KVPair's key.
-func GetSimulationLog(storeName string, cdcA, cdcB *codec.Codec, kvs []cmn.KVPair) (log string) {
+func GetSimulationLog(storeName string, sdr sdk.StoreDecoderRegistry, cdc *codec.Codec, kvs []cmn.KVPair) (log string) {
+	if len(kvs)%2 != 0 {
+		panic("KVPairs are not multiple of 2. There should be one for each app store")
+	}
```

Comment:
Feeding in all the kvs in one field is kinda confusing, which makes this error output also a bit confusing. I'd recommend:
``` golang
func GetSimulationLog(storeName string, sdr sdk.StoreDecoderRegistry, cdc *codec.Codec, kvAs, kvBs []cmn.KVPair) (log string) 
```
which would mean the error check could simply be:
```
if len(kvAs) != len(kvBs) {
    panic("Unequal sets of key-values to compare")
}
```